packages/app/provider/BaseProviderStack.tsx

```diff
@@ -23,24 +23,65 @@ export function BaseProviderStack({
   migrationState,
   children,
 }: PropsWithChildren<BaseProviderStackProps>) {
+  return (
+    <GlobalProviderStack>
+      <UIProviderStack tamaguiState={tamaguiState}>
+        <AppProviderStack migrationState={migrationState}>
+          {children}
+        </AppProviderStack>
+      </UIProviderStack>
+    </GlobalProviderStack>
+  );
+}
+
+/**
+ * Providers that depend on other providers farther up the stack
```

Comment:
This still feels pretty ambiguous, but I'm not sure we can provide much more clarity